src/pointcloud_to_laserscan_nodelet.cpp

```diff
@@ -69,6 +69,8 @@ void PointCloudToLaserScanNodelet::onInit()
   private_nh_.param<double>("range_min", range_min_, 0.0);
   private_nh_.param<double>("range_max", range_max_, std::numeric_limits<double>::max());
   private_nh_.param<double>("inf_epsilon", inf_epsilon_, 1.0);
+  private_nh_.param<std::string>("cloud_in", cloud_in_, "/hokuyo3d/hokuyo_cloud2");
+  private_nh_.param<std::string>("scan", scan_, "/hokuyo3d/scan");
```

Comment:
Topics should not be configurable by ROS parameter. They can be [remapped](https://wiki.ros.org/roslaunch/XML/remap).

Reply:
OK, I understand.
In addition, I want to remap from these topics by yaml files.
could you tell me some solution ?

Reply:
You can't remap topics from yaml files

Reply:
Ok, The way I said it was not good.
It' not that I want to remap topic, 
What I mean that I just want to set the "string of topic" by yaml, like [that](https://github.com/hku-mars/FAST_LIO/blob/main/src/laserMapping.cpp)

example:
nh.param<string>("common/lid_topic",lid_topic,"/livox/lidar");
nh.param<string>("common/imu_topic", imu_topic,"/livox/imu");

So, Would it be possible to check and merge it please ?

Reply:
You want to change the subscribed and published topics of the pointcloud_to_laserscan node. That functionality is already provided by remapping. Please use remapping in your launch file to change the topic names.

Reply:
OK. I concerned it seems to be this pkg's policy.
I'm grateful for your nice advice of your opinion.

I wanted to request easier method than remap. Then, I'd thought that we would be able to use this package more easily.

I 'll do my best.

Thank you.